embed/templates/scripts/run_grafana.sh.tpl

```diff
@@ -6,6 +6,15 @@ set -e
 DEPLOY_DIR={{.DeployDir}}
 cd "${DEPLOY_DIR}" || exit 1
 
+# try to compatible with tidb-lightning's dashboard,
+# the tidb-lightning dashboard's uid is not set,
+# after the cluster renamed, the old dashboard will not be replaced with the newly one.
```

Comment:
I checked old `dashboards/lightning.json` but there is no `uid` in this file. Is there something wrong?